webui/react/src/types.ts

```diff
@@ -62,12 +62,13 @@ export interface Telemetry {
 
 export enum ResourceType {
   CPU = 'CPU',
-  GPU = 'GPU',
+  CUDA = 'CUDA',
+  ROCM = 'ROCM',
```

Comment:
**nit:** the value is used for UI display. `ROCm` instead of `ROCM`?
```
ROCM = 'ROCm',
```

Reply:
it turned out it's also used in the code (e.g. `overview.ROCM` in `ClusterOverview.tsx`), not just UI display.

Reply:
ah gotcha